dev-packages/e2e-tests/test-applications/nuxt-3/tests/tracing.client.test.ts

```diff
@@ -0,0 +1,57 @@
+import { expect, test } from '@nuxt/test-utils/playwright';
+import { waitForTransaction } from '@sentry-internal/test-utils';
+import type { Span } from '@sentry/nuxt';
+
+test('sends a pageload root span with a parameterized URL', async ({ page }) => {
+  const transactionPromise = waitForTransaction('nuxt-3', async transactionEvent => {
+    return transactionEvent.transaction === '/test-param/:param()';
+  });
+
+  await page.goto(`/test-param/1234`);
+
+  const rootSpan = await transactionPromise;
+
+  expect(rootSpan).toMatchObject({
+    contexts: {
+      trace: {
+        data: {
+          'sentry.source': 'route',
+          'sentry.origin': 'auto.pageload.vue',
+          'sentry.op': 'pageload',
+          'params.param': '1234',
+        },
+        op: 'pageload',
+        origin: 'auto.pageload.vue',
+      },
+    },
+    transaction: '/test-param/:param()',
+    transaction_info: {
+      source: 'route',
+    },
+  });
+});
+
+test('sends component tracking spans when `trackComponents` is enabled', async ({ page }) => {
+  const transactionPromise = waitForTransaction('nuxt-3', async transactionEvent => {
+    return transactionEvent.transaction === '/client-error';
+  });
+
+  await page.goto(`/client-error`);
+
+  const rootSpan = await transactionPromise;
+  const errorButtonSpan = rootSpan.spans.find((span: Span) => span.description === 'Vue <<ErrorButton>>');
```

Comment:
heads-up: After merging https://github.com/getsentry/sentry-javascript/pull/13543/files, the component name should only be wrapped with `<Name>` and not `<<Name>>` anymore. Probably best to rebase to latest develop and double-check.